tests/test_collection.py

```diff
@@ -46,3 +68,9 @@ async def test_list_collection(client):
     assert collection_row is not None
     assert collection_row.text_embedding_model == "openai:text-embedding-ada-002"
     assert collection_row.text_distance_metric == "cosine"
+
+async def test_delete_collection(client, collection_fixture):
+    await delete_collection.asyncio(client=client, name=collection_fixture.collection1)
+
+    collections = await list_collections.asyncio(client=client)
+    assert collection_fixture.collection1 not in [c.name for c in collections]
```

Comment:
Should we test that deletion actually deletes documents / embeddings associated with the collection? Seems important.